content/outcomes/application-development-how-we-work/principles.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Our Foundational Principles"
+weight: 10
+layout: single
+team:
+  - VMware Tanzu Labs
+---
+
+Within Tanzu Labs, we work with an unpredictable assortment of companies, industries, governments, countries, cities, cultures, and people. **We help nearly every industry in the world refactor, reimagine, and build every kind of software product using any technology stack.** We work with countless unique embedded company cultures – some welcoming, some hostile, some ambivalent, yet nearly all have given us a mandate to help them rethink not only how they build products but how they build and manage teams. Sometimes they ask us to help them rethink their entire company culture. Sometimes they ask us to help them design their offices – really, their physical offices!
+
+Wow. That's a lot of anything and everything.
+
+In a world where we work with anyone, anywhere, anytime, upon anything, **what is our "north star" that guides how we approach so many diverse situations?**
+
+The answer is our **foundational principles**. Within Tanzu Labs, we've developed six principles which complement our [VMware EPIC<sub>2</sub> values](https://submissions.eng.vmware.com/epic2-achievement-awards-2020/pages/our-shared-values) with an eye towards how we work with clients and each other:
```

Comment:
Is this link behind the VPN? I get a failed DNS lookup when attempting to reach it

Reply:
Great catch, I thought this was a public marketing page. Will fix.

Reply:
Notable: the broken link checker did not catch this.

Reply:
Fixed.